README.md

```diff
@@ -50,6 +50,7 @@ For developers:
 
 - [Official repository on GitHub](https://github.com/MbinOrg/mbin)
 - [Matrix Space for discussions](https://matrix.to/#/#mbin:melroy.org)
+- [Unofficial magazine for discussions within fediverse](https://kbin.run/m/Mdev)
```

Comment:
`within the fediverse` would be better I think

Reply:
update the wording as suggested. I'm not that good at English